lib/surveyor/models/response_methods.rb

```diff
@@ -94,10 +109,14 @@ def to_formatted_s
         when :time
           time_value
         when :datetime
-          (read_attribute(:datetime_value).strftime(datetime_format) unless read_attribute(:datetime_value).blank?) || ''
```

Comment:
Whoever wrote this initial line is a monster.

Reply:
this entire gem is a monster mash